deploy/eck-elasticsearch/values.yaml

```diff
@@ -81,10 +129,355 @@ nodeSets:
+    # restartPolicy: "Always"
+
+    # If specified, the pod will be dispatched by specified scheduler. If not specified, the pod will be dispatched by default scheduler.
+    # schedulerName: ""
+
+    # SecurityContext holds pod-level security attributes and common container settings. Optional: Defaults to empty. See type description for default values of each field.
+    # See previously defined 'securityContext' within 'podTemplate' for all available fields.
+    # securityContext: {}
+
+    # ServiceAccountName is the name of the ServiceAccount to use to run this pod.
+    # https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+    # serviceAccountName: ""
+
+    # Share a single process namespace between all of the containers in a pod. When this is set containers will be able to view and signal processes from other containers in the same pod,
+    # and the first process in each container will not be assigned PID 1. HostPID and ShareProcessNamespace cannot both be set. Optional: Default to false.
+    # shareProcessNamespace: false
+
+    # If specified, the fully qualified Pod hostname will be "<hostname>.<subdomain>.<pod namespace>.svc.<cluster domain>". If not specified, the pod will not have a domainname at all.
+    # subdomain: ""
+
+    # Optional duration in seconds the pod needs to terminate gracefully. May be decreased in delete request. Value must be non-negative integer.
+    # The value zero indicates stop immediately via the kill signal (no opportunity to shut down).
+    # If this value is nil, the default grace period will be used instead. The grace period is the duration in seconds after the processes running
+    # in the pod are sent a termination signal and the time when the processes are forcibly halted with a kill signal.
+    # Set this value longer than the expected cleanup time for your process. Defaults to 30 seconds.
+    # terminationGracePeriodSeconds: 30s
+
+    # If specified, the pod's tolerations that will apply to all containers within the pod.
+    # https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/
+    # tolerations:
+    # - key: "node-role.kubernetes.io/elasticsearch"
+    #   effect: "NoSchedule"
+    #   operator: "Exists"
+
+    # TopologySpreadConstraints describes how a group of pods ought to spread across topology domains.
+    # Scheduler will schedule pods in a way which abides by the constraints. All topologySpreadConstraints are ANDed.
+    #
+    # These settings are generally applied within each `nodeSets[].podTemplate` field to apply to a specific Elasticsearch nodeset.
+    #
+    # https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-advanced-node-scheduling.html
+    # topologySpreadConstraints: {}
+
+    # List of volumes that can be mounted by containers belonging to the pod.
+    # https://kubernetes.io/docs/concepts/storage/volumes
+    # volumes: []
```

Comment:
I have reservations against dumping the whole PodTemplateSpec including docs here. Would a link to the Kubernetes doc not suffice?